### PR TITLE
feat(config): add config get & set subcommands

### PR DESCRIPTION
**Pull request by acburdine**
this is another approach to the feature added in #693 , using yargs subcommand support instead.

### CODE REVIEW

File: lib/commands/config/index.js [hunk 1]
```diff
@@ -5,6 +5,16 @@ const Command           = require('../../command');
 const advancedOptions   = require('./advanced');
 
 class ConfigCommand extends Command {
+    static configureSubcommands(commandName, commandArgs, extensions) {
+        return commandArgs.command({
```
Comment (vikaspotluri123) [inline lib/commands/config/index.js]: Do you know if it's possible to expose these options when running `ghost config help`?

File: lib/commands/config/index.js [hunk 2]
```diff
@@ -5,6 +5,16 @@ const Command           = require('../../command');
...
+            command: 'get <key>',
+            handler: (argv) => this._run(`${commandName} get`, argv, extensions)
```
Comment (vikaspotluri123) [inline lib/commands/config/index.js]: What exactly is the reasoning behind renaming the command here? Is it just for convenience or does yargs  somehow use it?

**Comment by vikaspotluri123** [comment]
I didn't think of chaining `.command` but now that I see it it makes a lot of sense!

File: lib/commands/config/index.js (hunk 1 above)
Comment (acburdine) [reply lib/commands/config/index.js]: Hmm...I didn’t test it but it should have. I’ll look into it.

File: lib/commands/config/index.js (hunk 2 above)
Comment (acburdine) [reply lib/commands/config/index.js]: So the reason that’s passed in is the debug output uses that command name to say the error occurred running command “x”. Probably not a _huge_ deal with these small commands, but it’s probably worth it to be consistent 🙂

Comment (vikaspotluri123) [reply lib/commands/config/index.js]: Got it - I guess it just feels weird to me that the command name is `ghost config (s|g)et`

File: lib/commands/config/index.js (hunk 1 above)
Comment (acburdine) [reply lib/commands/config/index.js]: Looks like all I needed to do was add a description 😁

**Comment by acburdine** [comment]
This should be good for a review now @AileenCGN  😄

**Comment by kirrg001** [comment]
I execute `ghost config` in an empty folder, it tells me

> Working directory is not a recognisable Ghost installation.

Makes sense. Then I execute `ghost config get` in an empty folder, it shows me the whole help output 🤔 

Feels a little inconsistent? 

---

Furthermore, if i set a value, i am not getting any feedback like "Successfully updated the config value".

**Comment by acburdine** [comment]
@kirrg001 the reason you're getting help output when you run `ghost config get` is because `ghost config get` requires a `key` to get a value for, and because you didn't pass one Yargs prints out help output.

**Comment by acburdine** [comment]
ok @kirrg001 - the log has been added :)

**Pull request merged**